packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -221,6 +221,10 @@ class ApplicationCommand extends Base {
    * {@link ApplicationCommandOptionType.Number} option
    * @property {number} [maxValue] The maximum value for an {@link ApplicationCommandOptionType.Integer} or
    * {@link ApplicationCommandOptionType.Number} option
+   * @property {number} [minLength] The minimum length for an {@link ApplicationCommandOptionType.String} option
```

Comment:
```suggestion
   * @property {number} [minLength] The minimum length for a {@link ApplicationCommandOptionType.String} option
```

Reply:
But it will appear as "The minimum length for a **ApplicationCommandOptionType.String** option"
Wouldn't it be "The minimum length for an **ApplicationCommandOptionType.String** option"? 🤔
If I'm not wrong, "an" is used when the word starts with a vowel sound.